docs/evm-sidechain/join-evm-sidechain-devnet.md

```diff
@@ -56,45 +56,68 @@ The first task is to initialize the node, which creates the necessary validator
 
 All these commands create your `~/.exrpd` (i.e `$HOME`) directory with subfolders `config/` and `data/`. In the `config` directory, the most important files for configuration are `app.toml` and `config.toml`.
 
+
 ## Genesis & Seeds
 
 1. Copy the Genesis File.
 
     Download the `genesis.json` file from here and copy it to the `config` directory: `~/.exrpd/config/genesis.json`. This is a genesis file with the chain-id and genesis accounts balances.
 
-        ```bash
-        wget https://raw.githubusercontent.com/Peersyst/xrp-evm-archive/main/poa-devnet/genesis.json ~/.exrpd/config/
-        ```
+    ```bash
+    wget https://raw.githubusercontent.com/Peersyst/xrp-evm-archive/main/poa-devnet/genesis.json ~/.exrpd/config/
+    ```
 
     Verify the genesis configuration file:
 
-        ```bash
-        exrpd validate-genesis
-        ```
+    ```bash
+    exrpd validate-genesis
+    ```
 
 2. Add Persistent Peer Nodes
 
     Set the [`persistent_peer`](https://docs.tendermint.com/master/tendermint-core/using-tendermint.html#persistent-peer)s field in `~/.exrpd/config/config.toml` to specify peers with which your node maintains persistent connections. You can retrieve them from the list of available peers on the archive repo ([https://raw.githubusercontent.com/Peersyst/xrp-evm-archive/main/devnet/peers.txt](https://raw.githubusercontent.com/Peersyst/xrp-evm-archive/main/devnet/peers.txt)).
 
     To get a list of entries from the `peers.txt` file in the `PEERS` variable, run the following command:
 
-        ```bash
-        PEERS=`curl -sL https://raw.githubusercontent.com/Peersyst/xrp-evm-archive/main/poa-devnet/peers.txt | sort -R | head -n 10 | awk '{print $1}' | paste -s -d, -`
-        ```
+    ```bash
+    PEERS=`curl -sL https://raw.githubusercontent.com/Peersyst/xrp-evm-archive/main/poa-devnet/peers.txt | sort -R | head -n 10 | awk '{print $1}' | paste -s -d, -`
+    ```
 
     Use `sed` to include them in the configuration. You can also add them manually:
 
-        ```bash
-        sed -i.bak -e "s/^persistent_peers *=.*/persistent_peers = \"$PEERS\"/" ~/.exrpd/config/config.toml
-        ```
+    ```bash
+    sed -i.bak -e "s/^persistent_peers *=.*/persistent_peers = \"$PEERS\"/" ~/.exrpd/config/config.toml
+    ```
+
+
+## Start a Node
+
+1. Start a node container.
+
+    ```bash
+    exrpd start
+    ```
+
+2. Provide Peersyst (<acarrera@peersyst.com>) with the key from this command:
```

Comment:
I suggest we use an email alias for this step.

Reply:
I reached out to him about this and he said to just use his own email for now. If it gets too crazy he'd set something up. Do you think we should ask him to set up something now?

Reply:
Checked with Ajit too - since it's early stage, we should be okay for now and we can work with them to update later.